x-pack/plugins/security_solution/public/common/components/header_actions/translations.ts

```diff
@@ -0,0 +1,144 @@
+/*
```

Comment:
Similar to the other comment, it would be great if we all acknowledge the problem of having the `security_solution/public/common/components` folder which contains a lot of non-common components. This generates codeowner review requests from area teams in many PRs even for changes that are not related to those teams.

For instance, I'm going through 112 files that GitHub says I own which are completely unrelated to the Rules area 😬 

I feel like most of the "common" components are actually related to certain features and could be moved to their subdomain folders if/when we create such.